E1a-ARSE/ALIN01/README.md

```diff
@@ -0,0 +1,171 @@
+# Test Instruction
+
+| Documentation Code    | Title                           | Exchange Code | Test Code | Author        | Data Owner | Version | Date       |
+| --------------------- | ------------------------------- | ------------- | --------- | ------------- | ---------- | ------- | ---------- |
+| IFC4.3AbRV_E1a_ALIN01 | Alignment Infrastructure Curves | E1a           | ALIN01    | Lars Wikström | FTIA       | 1.0     | 07.01.2022 |
+
+
+## Summary (Intent)
+
+With these instructions the infrastructure (e.g. Road) alignment exchange is established.
+
+| Info                         |                                       |
+| ---------------------------- | ------------------------------------- |
+| Number of alignment(s)       | 1                                     |
+| Properties of segments       | no                                    |
+| Horizontal layout            | Straight Line, Circular Arc, Clothoid |
+| Vertical layout              | Straight Line, Circular Arc           |
+| Geometric representation     | No                                    |
+| IFC reference file available | Yes                                   |
+
+The [Expected Results](#Expected-Results) section lists the material that will be used to assess the fulfilment of capabilities.
+
+:zap: **This is a test-driven process: refer to the [Validation Criteria](#Validation-Criteria) to understand what is required by the test** :zap:
+
+## Itemised Roots
+*This section lists the primary entities and concept templates being validated by this test instruction, these are listed to understand the scope of the data entities being addressed. this is the documentation of the data placed within BIMQ*
+
+The Test instruction addresses the import and export of the following IFC Entities & Concept Templates:
+
+:construction: under construction :construction:
+
+<details><summary>IFC Entities</summary>
+
+These entities represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD. **The scope of the test shall not be used as a definitive scope of the exchange, or of the entire MVD.**
+
+- Model setup
+  - IfcSite
+  - IfcRoad
```

Comment:
I'd list this in Spatial structure, separately from Model setup.

Reply:
Should be fixed now.